core-bundle/src/Resources/contao/classes/StyleSheets.php

```diff
@@ -1391,7 +1391,7 @@ public function exportStyleSheet(DataContainer $dc)
 		// Add the media query (see #7560)
 		if ($objStyleSheet->mediaQuery != '')
 		{
-			$objFile->append($objStyleSheet->mediaQuery . ' {');
+			$objFile->append('@media ' . str_replace('@media ', '', $objStyleSheet->mediaQuery) . ' {');
```

Comment:
Adding `@media` in the "Media query" field will break the front end, therefore we do not need the `str_replace()`.